stories/SQFormDialogStepper.stories.js

```diff
@@ -221,15 +236,7 @@ export const SQDialogStepperWithValidation = () => {
           alignItems: 'center',
           spacing: 6
         }}
-        initialValues={{
-          firstName: '',
-          lastName: '',
-          middleI: '',
-          newAccount: false,
-          accountID: '',
-          description: '',
-          age: ''
-        }}
+        initialValues={personalDateInitialValues}
```

Comment:
this should contain the `personalDataInitialValues` and the `accountDetailsInitialValues`
`initialValues={...personalDataInitialValues, ...accountDetailsInitialValues}`
The spelling on personalDataInitialValues is a little off too.  It says Date in the middle instead of Data